### DISCUSSION TITLE
2011/02/06/wordpress%E5%8F%AF%E4%BB%A5%E7%9B%B4%E6%8E%A5%E8%AE%BF%E9%97%AE%E4%BA%86%EF%BC%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#313] [Announcements]
# 2011/02/06/wordpress%E5%8F%AF%E4%BB%A5%E7%9B%B4%E6%8E%A5%E8%AE%BF%E9%97%AE%E4%BA%86%EF%BC%9F

刚刚用浏览器打开自己的blog，居然没翻墙，直接就打开了。Wordpress解锁了？

https://blog.qizhen.xyz/2011/02/06/wordpress%E5%8F%AF%E4%BB%A5%E7%9B%B4%E6%8E%A5%E8%AE%BF%E9%97%AE%E4%BA%86%EF%BC%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jiucheng Nie
2011年05月10日 02:25 编辑
我人就在美国，天天看你的blog，不用翻墙，哈哈哈哈~~~~

回复
one
2011年02月6日 14:23 编辑
我也很奇怪这个问题

回复
